src/term/mod.rs

```diff
@@ -792,6 +792,25 @@ impl Term {
             | Term::ParseError(_) => false,
         }
     }
+
+    /// Extract the static literal from string chunk. It only returns a `Some(..)`
+    /// when the term is a `Term::StrChunk` and all the chunks are `StrChunk::Literal(..)`
+    pub fn try_str_chunk_as_static_str(&self) -> Option<String> {
```

Comment:
Any reason not to put it inside `StrChunks` directly, and get rid of the `str_chunk` in the name?

Reply:
`StrChunks` is not a special type:  it is a term containing a `Vec<StrChunk>`

Reply:
Ah, it's a good reason, then :sweat_smile: